relayer/relayer/src/listeners/eth.rs

```diff
@@ -221,25 +218,40 @@ impl EthereumPausedListener {
         let address = eth_contract_address.parse::<Address>()?;
         let most_eth = Most::new(address, Arc::clone(&eth_connection));
 
-        select! {
-            cb_event = circuit_breaker_receiver.recv () => {
-                warn!(target: "EthereumPausedListener", "Exiting due to a circuit breaker event {cb_event:?}");
-                Ok(cb_event?)
-            },
+        loop {
+            debug!(target: "EthereumPausedListener", "Ping");
+
+            let is_paused_call = most_eth.paused();
+
+            select! {
+                cb_event = circuit_breaker_receiver.recv () => {
+                    warn!(target: "EthereumPausedListener","Exiting due to a circuit breaker event {cb_event:?}");
+                    return Ok(cb_event?);
+                },
 
-            _ = async {
-                loop {
+                is_paused = is_paused_call.call() => {
```

Comment:
isn't a call for transacting, doesn't this cost gas?

Reply:
`send` is for transacting, `call` for read-only/dry-run